docs/source/troubleshooting/macos.rst

```diff
@@ -11,3 +11,14 @@ If you get this error:
       .. code-block:: bash
 
          launchctl load -w /Library/LaunchAgents/org.freedesktop.dbus-session.plist
+
+No gamepad detected
+-------------------
+#. Verify that you've installed `VirtualHID <https://github.com/kotleni/VirtualHID-macOS/releases/latest>`_.
+
+Unable to create HID device
+---------------------------
+If you get this error:
+   `Gamepad: Unable to create HID device. May be fine if created previously.`
```

Comment:
This doesn't format properly (https://lizardbyte--756.org.readthedocs.build/projects/sunshine/en/756/troubleshooting/macos.html#unable-to-create-hid-device). Try 4 spaces before this line.

Reply:
Check it, last commit

Reply:
I think you misunderstood my statement.

```rst
Unable to create HID device
---------------------------
If you get this error:
    `Gamepad: Unable to create HID device. May be fine if created previously.`
```

Reply:
Lol, sorry. Now, all okay?

Reply:
Looks good now! https://lizardbyte--756.org.readthedocs.build/projects/sunshine/en/756/troubleshooting/macos.html#unable-to-create-hid-device

Reply:
🤗